osgar/drivers/realsense.py

```diff
@@ -0,0 +1,42 @@
+"""
+  RealSense2 OSGAR wrapper
+
+"""
+import pyrealsense2 as rs
+
+from osgar.node import Node
+from osgar.bus import BusShutdownException
+
+
+class RealSense(Node):
+    def __init__(self, config, bus):
+        super().__init__(config, bus)
+        bus.register('pose2d')
+        self.verbose = config.get('verbose', False)
+        self.pipe = None  # not initialized yet
```

Comment:
```suggestion
        self.pipeline = None  # not initialized yet
```
I was really confused like "what kind of pipe do we have here and why"...

Reply:
OK, this was copy & paste from one realsense2 example. I agree.

Reply:
fixed, renamed + test to catch all(?) usages